tools/gen_part_base_yaml.cc

```diff
@@ -8,16 +8,25 @@
 
```

Comment:
This looks like a standalone patch. Can you break this out into its own PR?

Reply:
This new option doesn't seem to have anything to do with generating part YAMLs and duplicates bittool's get_device_id action.

Reply:
Oh, I didn't read that carefully enough. Yeah, I'd agree this is probably out of scope